azurelinuxagent/daemon/resourcedisk/openwrt.py

```diff
@@ -0,0 +1,136 @@
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
+# Copyright 2018 Sonus Networks, Inc. (d.b.a. Ribbon Communications Operating Company)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.6+ and Openssl 1.0+
+#
+import os
+
+import azurelinuxagent.common.logger as logger
+import azurelinuxagent.common.utils.fileutil as fileutil
+import azurelinuxagent.common.utils.shellutil as shellutil
+import azurelinuxagent.common.conf as conf
+from azurelinuxagent.common.exception import ResourceDiskError
+from azurelinuxagent.daemon.resourcedisk.default import ResourceDiskHandler
+
+class OpenWRTResourceDiskHandler(ResourceDiskHandler):
+    def __init__(self):
+        super(OpenWRTResourceDiskHandler, self).__init__()
+        # Fase File System (FFS) is UFS
+        if self.fs == 'ufs' or self.fs == 'ufs2':
+            self.fs = 'ffs'
+
+    def reread_partition_table(self, device):
+        shellutil.run("hdparm -z {0}".format(device),
```

Comment:
Check the return code?

Reply:
No needs, this is just refreshing the table. Error can be safely ignored

Reply:
Do you agree? should I resolved this conversation?

Reply:
It is a good idea to log the error though, no?

Reply:
Good point, I will implement it.

Reply:
Fix have been done